templates/injector-psp.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and (eq (.Values.injector.enabled | toString) "true" ) (and (eq (.Values.global.enabled | toString) "true") (eq (.Values.global.pspEnable | toString) "true") ) }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "vault.fullname" . }}-agent-injector
+  labels:
+    app.kubernetes.io/name: {{ include "vault.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: docker/default,runtime/default
+    apparmor.security.beta.kubernetes.io/allowedProfileNames: runtime/default
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  runtime/default
+    apparmor.security.beta.kubernetes.io/defaultProfileName:  runtime/default
```

Comment:
I'm curious what you think about making this PodSecurityPolicy annotations a templated value, with these as the defaults. As I understand it, the apparmor profiles require a kernel module, which is not guaranteed to be present on any given kubernetes cluster nodes. So it might be useful for folks to adjust these accordingly.

Reply:
Good idea. I'll make the change.

Reply:
Sorry this took a while. I was in between moving homes. I have made the changes.

Reply:
@jasonodonnell @tvoran 

Sorry this took a while. I was in between moving homes. I have made the changes.